### ISSUE TITLE
bad compaler

### ISSUE THREAD

**Issue by KhristenkoYura** [cubicdaiya/ngx_small_light#66]
# nginx -t
nginx: [emerg] dlopen() "/etc/nginx/modules/ngx_http_small_light_module.so" failed (/etc/nginx/modules/ngx_http_small_light_module.so: undefined symbol: DrawSetStrokeColor) in /etc/nginx/nginx.cof:8
nginx: configuration file /etc/nginx/nginx.conf test failed

**Comment by cubicdaiya** [comment]
How do you setup ImageMagick?

**Comment by KhristenkoYura** [comment]
```
cd ~/ngx_small_light-source
./setup
cd ~/nginx-source
./auto/configure --modules-path=/etc/nginx/modules/ --add-dynamic-module=~/ngx_small_light-source
make
make install
```

**Comment by cubicdaiya** [comment]
Fixed in cb0dba11acff3a469390fef7a46904e05e48ef3c. Thank you for the report!
